base/applications/rapps/lang/pt-PT.rc

```diff
@@ -38,7 +38,7 @@ IDR_LINKMENU MENU
 BEGIN
     POPUP "popup"
     BEGIN
-        MENUITEM "A&brir endereço no navegador", ID_OPEN_LINK
+        MENUITEM "Abrir endereço n&o navegador", ID_OPEN_LINK
```

Comment:
or `&no`

Reply:
I used &O because that is used for en-US also, and does not collide.